react/src/components/molecules/ListingTable/index.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ClassNames from 'classnames';
+
+import Collapse from '../../animations/Collapse';
+
+import './style.css';
+
+class ListingTableItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({
+      open: !this.state.open
+    });
+  }
+
+  render() {
+    const {row} = this.props;
+    const visibleItems = row.visibleItems || 2;
+    const inlineAccordion = (row.items.length > visibleItems);
+    const rowClasses = ClassNames({
+      'ma__rich-text': true,
+      'js-accordion': inlineAccordion
+    });
+    const shownItems = row.items.slice(0, visibleItems);
+    const shownItemsContent = shownItems.map((item) => (<span className="ma__listing-table__data-item">{item}</span>));
+    const invisibleItems = (inlineAccordion) ? row.items.slice(visibleItems) : [];
+    const invisibleItemsContent = (invisibleItems.length) ? (
```

Comment:
Instead of the ternary where the else returns an empty string, I would use the common React pattern of `const invisibleItemsContent = (invisibleItems.length) && [...]`

Reply:
👍